sdcm/argus_results.py

```diff
@@ -61,7 +65,9 @@ class Meta:
         Columns = [
             ColumnMetadata(name="P90 read", unit="ms", type=ResultType.FLOAT),
             ColumnMetadata(name="P99 read", unit="ms", type=ResultType.FLOAT),
-            ColumnMetadata(name="duration", unit="HH:MM:SS", type=ResultType.DURATION)
+            ColumnMetadata(name="duration", unit="HH:MM:SS", type=ResultType.DURATION),
+            ColumnMetadata(name="Overview", unit="", type=ResultType.TEXT),
+            ColumnMetadata(name="QA dashboard", unit="", type=ResultType.TEXT),
```

Comment:
FYI @enaydanov is doing some work in #8594, so it might also need the alternator dashboard 

if we won't have data in it, it won't be shown, right ?

Reply:
> FYI @enaydanov is doing some work in https://github.com/scylladb/scylla-cluster-tests/pull/8594, so it might also need the alternator dashboard

It can be easily added, but I'd need example run to modify `send_result_to_argus` to match it to proper column.

> if we won't have data in it, it won't be shown, right ?

right